src/modules/esl-alert/core/esl-alert.less

```diff
@@ -0,0 +1,20 @@
+esl-alert {
```

Comment:
Do this style super abstract?
We can not rely on consumer X here, We should define something really abstract for consuming.